### PR TITLE
Create CODEOWNERS

### PR DESCRIPTION
**Pull request by gnikit**

### CODE REVIEW

**Comment by awvwgk** [comment]
Wouldn't it be better to have the @fortran-lang/vscode team as owner?

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,2 @@
+*      @gnikit
+*      @krvajal
```
Comment (awvwgk) [inline .github/CODEOWNERS:2]:
```suggestion
*      @fortran-lang/vscode
```

**Review by awvwgk — CHANGES_REQUESTED** [review]

**Comment by gnikit** [comment]
> Wouldn't it be better to have the @fortran-lang/vscode team as owner?

I wanted to ask you about this. Can we instead create an empty fortran-lang team?

**Comment by gnikit** [comment]
The owner should be the org not the ogs team (I think)

**Comment by gnikit** [comment]
It could circumvent and enforce some annoying branch protection rules from say automerging. I think you got a point. This PR was more of a reflex than a well thought out action. Let's scrap it all together and not implement it in the future (unless things change)

**Pull request closed without merging**